data/data-pipeline/data_pipeline/etl/sources/census_decennial/etl.py

```diff
@@ -147,6 +148,65 @@ def __init__(self):
             field_names.CENSUS_DECENNIAL_UNEMPLOYMENT_FIELD_2009
         )
 
+        # Race/Ethnicity fields
+        self.TOTAL_RACE_POPULATION_FIELD = "PCT086001"  # Total
+        self.ASIAN_FIELD = "PCT086002"  # Total!!Asian
+        self.BLACK_OR_AA_FIELD = "PCT086003"  # Total!!Black or African American
+        self.NATIVE_HI_OR_API_FIELD = (
+            "PCT086004"  # Total!!Native Hawaiian and Other Pacific Islander
+        )
+        self.WHITE_FIELD = "PCT086005"  # Total!!White
```

Comment:
Same comment on WHITE vs NON_HISPANIC_WHITE. 

Also: White is not the same as Non-Hispanic White. Census asks 1 question about race (Are you white, Black, [other options]?) And one about ethnicity (Are you Hispanic or non-Hispanic?). 

Many people identify as both white and Hispanic.

I'm not sure how we want to handle this -- I don't think 2010 decennial reports the crosstabs of both white and non-Hispanic. So this is probably good enough. But maybe we make a note?